rules/CodingStyle/ClassNameImport/UsedImports.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\CodingStyle\ClassNameImport;
```

Comment:
Could you move this to `ValueObjecŧ` namespace? We use it by convention for these objects to avoid mixing sevices and bare objects. E.g. CodingStyle/ValueObject

Reply:
done 👍